src/scenes/Header/HeaderSearch/HeaderSearch.tsx

```diff
@@ -0,0 +1,38 @@
+import { Input, InputAdornment, makeStyles } from '@material-ui/core';
+import { Search } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+
+const useStyles = makeStyles(() => ({
+  search: {
+    width: '100%',
+    maxWidth: '545px',
+    border: '1px solid #D1DADF',
+    color: 'textSecondary',
+    padding: '4px',
+    borderRadius: '6px',
+    '&:before': {
+      display: 'none',
+    },
+    '&:after': {
+      borderBottomColor: 'primary',
+    },
+  },
+}));
+
+export const HeaderSearch: FC = () => {
+  const classes = useStyles();
+
+  return (
+    <Input
```

Comment:
```suggestion
    <TextField variant="outlined"
```
or at least
```suggestion
    <OutlinedInput
```
if we aren't going to use the extra bits from TextField, like `label` & `helperText`.

Reply:
Decided to go with `OutlinedInput` since I can use the input adornment with it for the icon - 
![image](https://user-images.githubusercontent.com/4968115/81621834-763c8080-93bd-11ea-98fa-910aa5b2d9cf.png)